.github/ISSUE_TEMPLATE/build_error.md

```diff
@@ -0,0 +1,83 @@
+---
+name: Build error 
+about: Some package in Spack didn't build correctly  
+
+---
+
+*Thanks for taking the time to report this build failure. To proceed with the
+report please:*
+1. Title the issue "Installation issue: <name-of-the-package>".
+1. Provide the information required below.
+1. Clean the issue from the template instructions below when finished.
+
+We encourage you to try, as much as possible, to reduce your problem to the minimal example that still reproduces the issue. That would help us a lot in fixing it quickly and effectively!
+
+
+---
+
+### Steps to reproduce the issue
+
+```console
+$ spack install <spec> # Fill in the exact spec you are using
+... # and the relevant part of the error message
+```
+
+### Platform and user environment
+
+Please report here your OS:
+```commandline
+$ uname -a 
+Linux nuvolari 4.15.0-29-generic #31-Ubuntu SMP Tue Jul 17 15:39:52 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
+$ lsb_release -d
+Description:	Ubuntu 18.04.1 LTS
+``` 
+and, if relevant, post or attach:
+
+- `packages.yaml`
+- `compilers.yaml`
+
+to the issue
+
+### Additional information
+
+Sometimes the issue benefits from additional details. In these cases there are
+a few things we can suggest doing. First of all, you can post the full output of:
+```console
+$ spack spec --install-status <spec>
+...
+```
+to show people whether Spack installed a faulty software or if it was not able to
+build it at all. Spack as also commands to parse logs and report error and 
+warning messages:
+```console
+$ spack log-parse --show=errors,warnings <file-to-parse>
+```
+You might want to run this command on the `config.log` or any other similar file
+found in the stage directory:
+```console
+$ spack location -s <spec>
```

Comment:
(minor) may be worth noting that the user may still want to use the `find` command to look for config.log, since for some Autotools builds Spack will create a build subdirectory (by convention called `spack-build`).